src/lib/flows/create-donation/input-details.svelte

```diff
@@ -82,7 +96,7 @@
     createDonation(
       dispatch,
       recipientAccountId,
-      $context.selectedTokenAddress?.[0] ?? unreachable(),
+      selectedTokenAddress ?? unreachable(),
```

Comment:
Just want to point out that this is kind of similar to `shouldNeverHappen()` (or whatever it was called) in that it doesn't provide much help if it is indeed unreachable.

Reply:
true. it's somewhat different here in that we only use it on this repo if the value in question must be defined for this code to even be executed, which is what's the case here (createDonation called by confirm in wallet button which is disabled unless selectedTokenAddress is defined). But it's still a possible pitfall. Don't think it should be fixed in this PR but we could consider making the error message mandatory same way we're planning to on other repos